regression/cbmc/union10/main.c

```diff
@@ -0,0 +1,23 @@
+typedef struct
+{
+  union {
+    int access[256];
+  };
+} S1;
+
+typedef struct
+{
+  int c;
+} S2;
+
+static S1 (* const l[1]) = {((S1 *) ((0x50000000UL) + 0x00000) )};
```

Comment:
`0x00000`? Just `0`? Also why, won't cbmc just optimise `0x50000000 + 0` to eliminate the pointless +0?

Reply:
Maybe it's just me, but this declaration baffled me -- the bracketing around `* const l[1]` suggested a function? Don't know if there's a more obvious way to write the type "array of const S1*s" though.